_inc/client/at-a-glance/search.jsx

```diff
@@ -7,6 +7,7 @@ import { connect } from 'react-redux';
 import { translate as __ } from 'i18n-calypso';
```

Comment:
Do we want to tweak `Jetpack Search is powering search on your site.` is this file?